config/samples/kustomization.yaml

```diff
@@ -1,4 +1,6 @@
 # SPDX-License-Identifier: MIT
 # SPDX-FileCopyrightText: (c) Copyright 2023 Advanced Micro Devices, Inc.
+
+# Bundled into CSV's `alm-examples`
 resources:
-- base
+- onload/overlays/in-cluster-build-ocp
```

Comment:
Do you reckon it's a good timing to enable SFC (uncomment `sfc: {}`) in the overlay file?